File: example/src/main/java/jp/bglb/bonboru/flux/example/reducer/MyReducer.java
package jp.bglb.bonboru.flux.example.reducer;

import jp.bglb.bonboru.flux.example.action.ActionTypes;
import jp.bglb.bonboru.flux.reducer.Reducer;

/**
 * Created by tmasuda on 2016/04/24.
 */
public abstract class MyReducer<T> extends Reducer<T, ActionTypes> {
}

File: kotlinexample/src/main/kotlin/com/example/kotlinexample/reducer/MyReducer.kt
package com.example.kotlinexample.reducer

import com.example.kotlinexample.action.MyActionType
import jp.bglb.bonboru.flux.reducer.Reducer

/**
 * Created by tetsuya on 2016/06/25.
 */
abstract class MyReducer<T>() : Reducer<T, MyActionType>()
